app/src/main/java/com/example/cleanarchtest/di/UseCaseModule.kt

```diff
@@ -1,10 +1,13 @@
 package com.example.cleanarchtest.di
 
 import com.example.domain.model.PostUseCases
+import com.example.domain.useCase.ApolloGetCountryUseCase
 import com.example.domain.useCase.GetAllPostUseCase
 import org.koin.dsl.module
 
 val UseCaseModule = module {
-    single { GetAllPostUseCase(get()) }
-    single { PostUseCases(get()) }
+    single { GetAllPostUseCase(postRepository = get()) }
+    single { PostUseCases(getAllPostUseCase = get()) }
+
+    single { ApolloGetCountryUseCase(apolloCountryRepository = get()) }
```

Comment:
We should remove the prefix `Apollo` from both the class and parameter name as we don't want to make these members dependent upon a library that we might change in the future.